Extension/src/LanguageServer/client.ts

```diff
@@ -71,7 +71,12 @@ interface OutputNotificationBody {
 
 interface InactiveRegionParams {
     uri: string;
-    ranges: vscode.Range[];
+    ranges: InputRange[];
+}
+
+interface InputRange {
+    start: { line: number, character: number };
```

Comment:
>, [](start = 25, length = 1)

nit: should be a semi-colon.  (surprised the linter didn't pick this up)

Reply:
Noted for the future!